.travis.yml

```diff
@@ -5,9 +5,10 @@ language: ruby
 sudo: false
 
 rvm:
-  - &ruby1 2.3.1
-  - &ruby2 2.2.5
-  - &ruby3 2.1.9
+  - &ruby1 2.4.0
+  - &ruby2 2.3.1
+  - &ruby3 2.2.5
+  - &ruby4 2.1.9
```

Comment:
How about bumping previous versions? Current versions are 2.3.3, 2.2.6, and 2.1.10. https://www.ruby-lang.org/en/downloads/